.github/CODEOWNERS

```
@@ -0,0 +1,358 @@
+/clang-tools-extra/docs/clang-tidy/ @llvm/pr-subscribers-clang-tidy
+/clang-tools-extra/test/clang-tidy/ @llvm/pr-subscribers-clang-tidy
+
+# llvm-mca
+/llvm/tools//llvm-mca/ @llvm/pr-subscribers-llvm-mca
+/llvm/include//llvm/MCA/ @llvm/pr-subscribers-llvm-mca
+/llvm/lib/MCA/ @llvm/pr-subscribers-llvm-mca
+
+# vectorizers
+/llvm/lib/Transforms/Vectorize @llvm/pr-subscribers-vectorizers
+
+# clang
+/clang/ @llvm/pr-subscribers-clang
+
+# testing-tools
+/llvm/include//llvm/FileCheck/ @llvm/pr-subscribers-testing-tools
+/llvm/lib/FileCheck/ @llvm/pr-subscribers-testing-tools
+/llvm/test/FileCheck/ @llvm/pr-subscribers-testing-tools
+/llvm/unittests/FileCheck/ @llvm/pr-subscribers-testing-tools
+/llvm/utils/lit/ @llvm/pr-subscribers-testing-tools
+/llvm/utils/split-file @llvm/pr-subscribers-testing-tools
+/llvm/utils/not @llvm/pr-subscribers-testing-tools
+/llvm/utils/count @llvm/pr-subscribers-testing-tools
+/llvm/utils/FileCheck/ @llvm/pr-subscribers-testing-tools
+/llvm/docs/CommandGuide/FileCheck.rst @llvm/pr-subscribers-testing-tools
+/llvm/docs/CommandGuide/lit.rst @llvm/pr-subscribers-testing-tools
+/llvm/docs/TestingGuide.rst @llvm/pr-subscribers-testing-tools
+/llvm/test/Other/FileCheck-space.txt @llvm/pr-subscribers-testing-tools
+
+# debug-info
+/llvm/include/llvm/DebugInfo @llvm/pr-subscribers-debug-info
+/llvm/lib/DebugInfo @llvm/pr-subscribers-debug-info
+/llvm/tools/dsymutil @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-debuginfo-analyzer @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-dwarfdump @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-dwarfutil @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-dwp @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-gsymutil @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-pdbutil @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-debuginfod @llvm/pr-subscribers-debug-info
+/llvm/tools/llvm-debuginfod-find @llvm/pr-subscribers-debug-info
+/llvm/lib/CodeGen/AsmPrinter @llvm/pr-subscribers-debug-info
+/clang/lib/CodeGen/CGDebugInfo.cpp @llvm/pr-subscribers-debug-info
+/llvm/include/llvm/BinaryFormat/Dwarf.* @llvm/pr-subscribers-debug-info
+/llvm/test/DebugInfo @llvm/pr-subscribers-debug-info
+/llvm/test/tools/dsymutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfo-analyzer @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfod @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-debuginfod-find @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwarfdump @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwarfutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-dwp @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-gsymutil @llvm/pr-subscribers-debug-info
+/llvm/test/tools/llvm-pdbuti @llvm/pr-subscribers-debug-info
+/llvm/lib/IR/Debug*.cpp @llvm/pr-subscribers-debug-info
+/llvm/include/llvm/IR/Debug*.h @llvm/pr-subscribers-debug-info
+
+# github-workflow
+/.github/workflows/ @llvm/pr-subscribers-github-workflow
+
```

Can we add these for flang

```suggestion
/flang/ @llvm/ps-subscribers-flang
/flang/runtime/ @llvm/pr-subscribers-flang-runtime
/flang/**/Parser @llvm/pr-subscribers-flang-parser
/flang/**/Evaluate/ @llvm/pr-subscribers-flang-semantics
/flang/**/Semantics/ @llvm/pr-subscribers-flang-semantics
/flang/**/Lower/ @llvm/pr-subscribers-flang-fir-hlfir
/flang/**/Optimizer/ @llvm/pr-subscribers-flang-fir-hlfir
/flang/**/CodeGen/ @llvm/pr-subscribers-flang-codegen
```